Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3193[Validation] Avg Loss: 0.2870, Avg Dice Score: 0.8983 100% ||[Finetuning] Epoch: 2 Avg Loss: 0.3104[Validation] Avg Loss: 0.2916, Avg Dice Score: 0.8850100%[Finetuning] Epoch: 3 Avg Loss: 0.3054[Validation] Avg Loss: 0.2890, Avg Dice Score: 0.8884100%[Finetuning] Epoch: 4 Avg Loss: 0.3017[Validation] Avg Loss: 0.2901, Avg Dice Score: 0.8866100%[Finetuning] Epoch: 5 Avg Loss: 0.2971[Validation] Avg Loss: 0.2877, Avg Dice Score: 0.8906100%[Finetuning] Epoch: 6 Avg Loss: 0.2956[Validation] Avg Loss: 0.2856, Avg Dice Score: 0.8964100%[Finetuning] Epoch: 7 Avg Loss: 0.2936[Validation] Avg Loss: 0.2879, Avg Dice Score: 0.8914100%[Finetuning] Epoch: 8 Avg Loss: 0.2928[Validation] Avg Loss: 0.2823, Avg Dice Score: 0.8982100%[Finetuning] Epoch: 9 Avg Loss: 0.2914[Validation] Avg Loss: 0.2871, Avg Dice Score: 0.8926100%[Finetuning] Epoch: 10 Avg Loss: 0.2896143/143 [00:13<00:00, 10.73it/s]143/143 [00:13<00:00, 10.63it/s]143/143 [00:13<00:00, 10.66it/s]143/143 [00:13<00:00, 10.48it/s]143/143 [00:13<00:00, 10.57it/s]143/143 [00:13<00:00, 10.51it/s]143/143 [00:13<00:00, 10.63it/s]143/143 [00:13<00:00, 10.73it/s]143/143 [00:13<00:00, 10.66it/s]143/143 [00:13<00:00, 10.65it/s][Validation] Avg Loss: 0.2852, Avg Dice Score: 0.8951 /home/mikhelson/MedImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to 'self.log()' but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'TrainerTest Results - Average Loss: 0.6237, Average Dice Score: 0.7132